src/views/virtualmachines/actions/actions.ts

```diff
@@ -116,22 +110,3 @@ export const deleteVM = async (vm: V1VirtualMachine) => {
     resource: vm,
   });
 };
-
-export const rollbackStorageMigration = async (vm: V1VirtualMachine) => {
-  if (isEmpty(getUpdateStrategy(vm))) return;
-
-  const patchData: Patch[] = createRollbackPatchData(vm);
-
-  patchData.push({
-    op: 'remove',
-    path: '/spec/updateVolumesStrategy',
```

Comment:
it seems we are changing the rollback procedure - is it a new change in 4.19?

Reply:
Yes we need that as we don't want to use `updateVolumesStrategy` anymore. Its a storage migration method that do not work for all cases